src/esp/sensor/SensorFactory.cpp

```diff
@@ -1,22 +1,26 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
 #include "esp/sensor/SensorFactory.h"
 
 #include "esp/scene/SceneNode.h"
 #include "esp/sensor/CameraSensor.h"
+#include "esp/sensor/Sensor.h"
 
 namespace esp {
 namespace sensor {
-sensor::SensorSuite SensorFactory::createSensors(
-    scene::SceneNode& node,
-    const sensor::SensorSetup& sensorSetup) {
-  sensor::SensorSuite sensorSuite = sensor::SensorSuite();
+void SensorFactory::createSensors(scene::SceneNode& node,
+                                  const sensor::SensorSetup& sensorSetup) {
   for (const SensorSpec::ptr& spec : sensorSetup) {
     scene::SceneNode& sensorNode = node.createChild();
     // VisualSensor Setup
     if (spec->isVisualSensorSpec()) {
       if (spec->sensorSubType == SensorSubType::Orthographic ||
           spec->sensorSubType == SensorSubType::Pinhole) {
-        sensorSuite.add(CameraSensor::create(
-            sensorNode, std::dynamic_pointer_cast<CameraSensorSpec>(spec)));
+        sensor::CameraSensor::create(
+            sensorNode,
+            std::dynamic_pointer_cast<sensor::CameraSensorSpec>(spec));
```

Comment:
This is a good one. @mosra 
You will have to register this feature in that node. Try one of the following:
```
sensorNode.addFeature<sensor::CameraSensor>(std::dynamic_pointer_cast<sensor::CameraSensorSpec>(spec));
```
Or this one:
```
       new sensor::CameraSensor{
           sensorNode,
           std::dynamic_pointer_cast<sensor::CameraSensorSpec>(spec)};
```

Reply:
Thanks!